.editorconfig

```diff
@@ -1,15 +1,13 @@
 root = true
 
 [*]
+indent_style = space
```

Comment:
😁

Reply:
I use standard as linter, one less config to worry about. 😁

Reply:
But, mah semi-colons! 😄 (I actually like semis)

Reply:
I think there's a "semistandard", standard but with semis. I'll look into that, as well as prettier. 👍 As long as it's one less thing in the repo, it's totally fine by me.